src/components/landingPage/LandingPage.tsx

```diff
@@ -1,10 +1,20 @@
 import React from "react";
+import "./LandingPage.scss";
+import { Link } from "react-router-dom";
 
 export const LandingPage: React.FunctionComponent = () => {
   return (
     <main>
-      <h1>Landing Page</h1>
-      <p>Landing page</p>
+      <img src="/images/image-of-rovers-on-mars.jpg" alt="images of rovers on Mars" className="headline-image"/>
+      <h2>Welcome, trainee astronaut, to your Mission to Mars!</h2>
+      <p>Here you will learn about the Red Planet and explore its surface.</p>
+      <p>When the training is complete, we will test if you are ready to be an astronaut!</p>
+
+      <div className="landing-page-button-menu">
```

Comment:
Very nice class name! I think we can follow [BEM naming](https://codeburst.io/understanding-css-bem-naming-convention-a8cca116d252) a little more closely though 😉 

How about `landing-page__button-menu`? Here the "block" is the landing page and the "element" is the button menu